daal4py/INSTALL.md

```diff
@@ -3,8 +3,9 @@ daal4py is easily built from sources with the majority of the necessary prerequi
 
 The build-process (using setup.py) happens in 3 stages:
 1. Creating C++ and cython sources from oneDAL C++ headers
-2. Running cython on generated source
-3. Compiling and linking
+2. Building onedal python interfaces by cmake and pybind11
+3. Running cython on generated source
+4. Compiling and linking
```

Comment:
```suggestion
2. Building oneDAL Python interfaces via cmake and pybind11
3. Running Cython on generated sources
4. Compiling and linking them
```

last item requires you to specify what are you compiling and linking. I put `them` there to refer to `generated sources` from the previous item but it might be incorrect.

Reply:
You are right, thanks